docs/HISTORY.txt

```diff
@@ -5,7 +5,8 @@ CHANGELOG
 1.4.2 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Remove workflows.
```

Comment:
I think we need to explain why we do this. Maybe:

```
Remove default workflows, since workflows should be done on integration level.
The default workflow is now used for billboard types.
```